backend/chainlit/server.py

```diff
@@ -112,8 +120,7 @@ async def watch_files_for_changes():
         os._exit(0)
 
 
-root_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-build_dir = os.path.join(root_dir, "frontend/dist")
+build_dir = os.path.join(PACKAGE_ROOT, "frontend/dist")
```

Comment:
Same feedback as my other comment
```suggestion
build_dir = os.path.join(PACKAGE_ROOT, "frontend", "dist")
```

Reply:
Good catch!